src/server/server_family.cc

```diff
@@ -923,7 +923,7 @@ void PrintPrometheusMetrics(const Metrics& m, StringResponse* resp) {
                        &command_metrics);
     for (const auto& [name, stat] : m.cmd_stats_map) {
       const auto calls = stat.first;
-      const auto duration_seconds = stat.second * 0.001;
+      const double duration_seconds = stat.second * 0.001;
```

Comment:
![hackerman](https://ih1.redbubble.net/image.980358996.2852/flat,750x,075,f-pad,750x1000,f8f8f8.jpg) :rofl:

Reply:
Not sure what you meant, I was referring to using `1e-9` that was there before :)
(I think it's readable, just thought it would be funny to add this meme. anyway, please ignore)

Reply:
It's not clear!!!

Reply:
`1e-9` is not here. I do not understand this meme 🤷🏼

Reply:
please ignore, never mind :)